Modules/HypnokinkModule/Events/HK_VionMeetingEvent.gd

```diff
@@ -0,0 +1,25 @@
+extends EventBase
+
+func _init():
+	id = "HK_VionMeetingEvent"
+
+func registerTriggers(es):
+	es.addTrigger(self, Trigger.EnteringRoom, "cellblock_lilac_nearcell")
```

Comment:
Old non-lilac players won't find this (I didn't x3). Maybe there is a better way to introduce him? Like I did with Jacki, you would run into her eventually. Also, dunno if making it so you can find him right after starting the game is a good idea. Do you want him to potentially be the first named npc that players interact with? x3 (after intro I mean). Considering he has a somewhat specific kink that not everyone would be into

Reply:
I'll rejig it a bit - probably make the cell unsearchable the first day (or however long the player takes to at least visit the mineshafts) and have Vion walk by a few days in to direct the player towards the cellblock with some hint text.

Reply:
Made it require the player to be at least four days in AND have actually bothered to visit the mines. Vion also shows up in a super-short random encounter that directs the player towards the cellblock.